lib/e2e/src/assert.rs

```diff
@@ -1,28 +1,23 @@
-use alloy::{sol_types::SolError, transports::RpcError};
+use alloy::sol_types::SolError;
 
-pub trait Assert<E> {
+pub trait ErrorExt<E> {
     /// Asserts that current error result corresponds to the typed abi-encoded
```

Comment:
No `assert` here; may be misleading.

Reply:
Yeah, I'll rework this a bit, and will probably move this to a separate file.

Reply:
sure, ping me when I should re-review it.